lib/chef/win32/api.rb

```diff
@@ -117,6 +117,7 @@ def self.extended(host)
         host.typedef :uint32,  :LCID # Locale identifier. For more information, see Locales.
         host.typedef :uint32,  :LCTYPE # Locale information type. For a list, see Locale Information Constants.
         host.typedef :uint32,  :LGRPID # Language group identifier. For a list, see EnumLanguageGroupLocales.
+        host.typedef :pointer, :LMSTR
```

Comment:
"A LMSTR is a 32-bit pointer to a null-terminated string of 16-bit Unicode characters."